tests/k6/tests/graphql/performance/graphqlCommonFunctions.js

```
@@ -0,0 +1,284 @@
+    default:
+      return createBodyForAllDialogsForParty(endUser);
+  }
+}
+
+export function createBodyForMultiParties(parties, type) {
+  switch (type) {
+    case "getAllDialogsForParties":
+      return createBodyForAllDialogsForParties(parties);
+    case "getAllDialogsForPartiesFts":
+      return createBodyForAllDialogsForPartiesFts(parties);
+    case "getAllDialogsForParties":
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)